contracts/cheddar_5x5_tic_tac_toe/src/internal.rs

```diff
@@ -41,19 +41,20 @@ impl Contract {
                 log!(
                     "GameId: {}. Game duration expired. Required:{} Current:{} ",
                     game_id,
-                    self.max_game_duration,
+                    self.max_game_duration_sec,
                     ts - game.initiated_at
                 );
             }
         }
-        self.last_update_timestamp = ts;
+        self.last_update_timestamp = nano_to_sec(ts);
     }
```

Comment:
do we need this function?

Reply:
This is the current implementation and it is being used. Once we divide the make_available and deposit_transfer and remember the balance for each user we can delete it. We have a tasks for it